src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -655,6 +667,69 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
             }
         }
 
+        /// <summary>
+        /// Creates a dictionary which maps a request method name to its clr method name via <see cref="JsonRpcMethodAttribute" /> value.
+        /// </summary>
+        /// <param name="target">Object to reflect over and analyze its methods.</param>
+        /// <returns>Dictionary which maps a request method name to its clr method name.</returns>
+        private void GetRequestMethodToClrMethodMap(object target)
+        {
+            if (target != null)
+            {
+                for (TypeInfo t = target.GetType().GetTypeInfo(); t != null; t = t.BaseType?.GetTypeInfo())
+                {
+                    foreach (MethodInfo method in t.DeclaredMethods)
+                    {
+                        var attribute = (JsonRpcMethodAttribute)method.GetCustomAttribute(typeof(JsonRpcMethodAttribute));
+                        var attributeValue = attribute == null ? null : attribute.Name;
```

Comment:
Tip: 

```csharp
var attributeValue = attribute == null ? null : attribute.Name;
```
is equivalent to
```csharp
var attributeValue = attribute?.Name;
```

And as the type is not stated elsewhere on this line, using `string` instead of `var` would be preferable here.